test/unittests/skills/test_utterance_intents.py

```diff
@@ -51,13 +59,19 @@ def test_padatious_intent(self):
         self.assertEqual(intent.matches, {'thing': 'Mycroft'})
 
         # fuzzy regex match - success
-        intent = intent_service.calc_intent("tell me everything about Mycroft", "en-US")
+        utterance = "tell me everything about Mycroft"
+        intent = intent_service.calc_intent(utterance, "en-US")
         self.assertEqual(intent.name, "test2")
-        # TODO - why are extracted entities lower case ???
-        # i think case depends on padaos vs padatious matching internally
+
+        # case depends on padaos vs padatious matching internally
         # padaos (exact matches only) -> keep case
```

Comment:
these comments are from before padacioso replaced padaos (only exact matches)

does this still apply? i think that if using padatious directly the casing is inconsistent too depending on its internal usage of padaos

Reply:
padatious doesn't seem to handle case consistently, but padacioso does so I think it makes sense to leave some note